src/MongoDB.Driver/AggregateFluent.cs

```diff
@@ -146,6 +147,17 @@ public override IAggregateFluent<TResult> Limit(int limit)
             return WithPipeline(_pipeline.Lookup(foreignCollection, localField, foreignField, @as, options));
         }
 
+        public override IAggregateFluent<TNewResult> Lookup<TForeignDocument, TAsElement, TAs, TNewResult>(
+            IMongoCollection<TForeignDocument> foreignCollection,
+            BsonDocument let,
+            PipelineDefinition<TForeignDocument, TAsElement> pipeline,
```

Comment:
In some other files we call this parameter "lookupPipeline" to avoid a name collision with an existing "pipeline" parameter.

Let's be consistent and name this parameter "lookupPipeline" everywhere (including "pipeline" variables in test code).

Reply:
This is due to the fact that the `Lookup` method in `PipilineDefinitionBuilder` already has method parameter with `pipeline` name (which is extensions parameter: ``` this PipelineDefinition<TInput, TIntermediate> pipeline,```). But I can rename the extensions parameter and stay lookup parameter as in other methods